bmwqemu.pm

```diff
@@ -975,6 +977,16 @@ sub _scale_timeout {
     return $timeout * ($vars{TIMEOUT_SCALE} // 1);
 }
 
+# just a random string useful for pseudo security or temporary files
+sub random_string {
+    my ($count) = @_;
+    $count //= 4;
```

Comment:
just nitpick, but if $count is 0, this will leave it untouched. Do you really want random_string to produce empty string (still random though :) )?
I would prefer ||=